CHANGES.rst

```diff
@@ -17,8 +17,6 @@ jump detection
 documentation
 -------------
 
-- Update jump step docs [#1035]
```

Comment:
Was this change intended?

Reply:
This went in Dec 7, I think I just moved the text. 
We used to keep the change log in alphabetical order. I think I'll just give up.